xml/System.IO/FileInfo.xml

```diff
@@ -2198,13 +2198,16 @@ The following example demonstrates moving a file to a different location and ren
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Returns the path as a string.</summary>
+        <summary>Returns the path as a string. Use the <see cref="P:System.IO.FileInfo.Name" /> property for the full path.</summary>
         <returns>A string representing the path.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- The string returned by the <xref:System.IO.FileInfo.ToString%2A> method represents path that was passed to the constructor.  When you create a <xref:System.IO.FileInfo> object using the constructors, the <xref:System.IO.FileInfo.ToString%2A> method returns the fully qualified path.  However, there are cases where the string returned by the <xref:System.IO.FileInfo.ToString%2A> method does not represent the fully qualified path.  For example, when you create a <xref:System.IO.FileInfo> object using the <xref:System.IO.DirectoryInfo.GetFiles%2A> method, the <xref:System.IO.FileInfo.ToString%2A> method does not represent the fully qualified path.  
+## Remarks
+
+> [!IMPORTANT]
+
+> The string returned by the <xref:System.IO.FileInfo.ToString%2A> method represents path that was passed to the constructor. When you create a <xref:System.IO.FileInfo> object using the constructors, the <xref:System.IO.FileInfo.ToString%2A> method returns the fully qualified path. However, there are cases where the string returned by the <xref:System.IO.FileInfo.ToString%2A> method does not represent the fully qualified path. For example, when you create a <xref:System.IO.FileInfo> object using the <xref:System.IO.DirectoryInfo.GetFiles%2A> method, the <xref:System.IO.FileInfo.ToString%2A> method does not represent the fully qualified path. To ensure consistent behavior and explicit intent, call `ToString` on the <xref:System.IO.FileInfo.Name> of the current <xref:System.IO.FileInfo> instance.
```

Comment:
```suggestion
> The string returned by the <xref:System.IO.FileInfo.ToString%2A> method represents the path that was passed to the <xref:System.IO.FileInfo> constructor. Instead of calling the `ToString` method, you should retrieve the value of either of the following properties, depending on your intent:
- <xref:System.IO.FileInfo.Name>, to get the name of the file, without any parent path information.
- <xref:System.IO.FileInfo.FullName>, to get the fully qualified path of the file.
```